sdk/personalizer/Azure.AI.Personalizer/tests/Azure.AI.Personalizer.Tests.csproj

```diff
@@ -6,16 +6,16 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="RL.Net" VersionOverride="0.2.0" />
+    <PackageReference Include="RL.Net" />
```

Comment:
nit shouldn't the test project get this from Azure.AI.Personalizer

Reply:
Yes, it should.   I'll put out a follow-up PR for this shortly.